pom.xml

```diff
@@ -259,6 +259,18 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-jar-plugin</artifactId>
+					<version>3.0.2</version>
+					<executions>
+						<execution>
+							<goals>
+								<goal>test-jar</goal>
```

Comment:
does this mean that we build a test-jar for each module? Wouldn't it be sufficient to add the configuration to the modules that actually want to build a test-jar?

Reply:
I hope not, as it is in the pluginManagement section. 
And I don't see the scandium test-jar after building with tests.

Reply:
> Wouldn't it be sufficient to add the configuration to the modules that actually want to build a test-jar?

Sure, could the executions be removed from the parent POM and be copied twice (or where ever we will create a test-jar). But I'm not sure, why this copies are intended to have advantages.

Reply:
So, though I would add some comments about the extra "profile", should we go for copies of the goal test-jar or should we keep it without copies?

Reply:
My personal preference would be to add the executions only to those modules that actually want to build a test-jar. Doing so is actually not very common anyways (at least I haven't been working on any other project where this has been done). To me it makes it much more transparent where and why we want/need to create test-jars (assuming that we document the execution configuration properly in the modules' POM files).

Reply:
OK, I will mode the goal test-jar